src/bidiServer/MapperCdpConnection.ts

```diff
@@ -143,34 +143,22 @@ export class MapperServerCdpConnection {
 
     const browserClient = await cdpConnection.createBrowserSession();
 
-    // Run mapper in the first open tab.
-    const targets = (await cdpConnection.sendCommand(
-      'Target.getTargets',
-      {},
-    )) as Protocol.Target.GetTargetsResponse;
-    const mapperTabTargetId = targets.targetInfos.filter(
-      (target) => target.type === 'page',
-    )[0]!.targetId;
+    const {targetId: mapperTargetId} = await browserClient.sendCommand(
+      'Target.createTarget',
+      {
+        url: 'about:blank#MAPPER_TARGET',
+        hidden: !verbose,
```

Comment:
Using verbose can be a bit unstable, as there can be issues reproducible only in hidden targets. If so, we can add another flag for running Mapper in a visible Tab if needed.